BGL/include/CGAL/boost/graph/Euler_operations.h

```diff
@@ -1754,6 +1754,72 @@ flip_edge(typename boost::graph_traits<Graph>::halfedge_descriptor h,
   set_halfedge(foh,oh,g);
 }
 
+/// \todo document me
```

Comment:
Even in case this functions remains in this pull request, make a separated smalll feature for it as soon as you have written the documentation.

Reply:
It is actually a function, that wasn't even used by the kinetic package, when I took it. So if it is interesting to keep, it should certainly be removed from this package.